.github/workflows/ci.yml

```diff
@@ -106,8 +106,7 @@ jobs:
         run: |
           sudo dpkg --add-architecture i386
           sudo apt update || true
-          sudo apt install lib32z1 git pkg-config libssl-dev:i386 libssl-dev zlib1g-dev:i386 curl gcc libdbus-1-dev
-          sudo apt install gcc:i386
+          sudo apt install zlib1g-dev:i386
```

Comment:
I guess we dont need all the dev deps if were running the binary.

If we had desktop notifs on we would almost certainly segfault though lmao.